hadoop-common-project/hadoop-registry/pom.xml

```diff
@@ -221,6 +221,7 @@
       <groupId>org.apache.maven.plugins</groupId>
       <artifactId>maven-surefire-plugin</artifactId>
       <configuration>
+        <testFailureIgnore>true</testFailureIgnore>
```

Comment:
temporary?

Reply:
Yes temporary, to continue execution to next module when previous module have unit test failures. (In case of multi module patch)